site/src/Librecores/ProjectRepoBundle/RepoCrawler/GitRepoCrawler.php

```diff
@@ -2,14 +2,18 @@
 
 namespace Librecores\ProjectRepoBundle\RepoCrawler;
 
+use Doctrine\Common\Persistence\ObjectManager;
 use Librecores\ProjectRepoBundle\Entity\Commit;
 use Librecores\ProjectRepoBundle\Entity\GitSourceRepo;
 use Librecores\ProjectRepoBundle\Entity\LanguageStat;
 use Librecores\ProjectRepoBundle\Entity\ProjectRelease;
+use Librecores\ProjectRepoBundle\Entity\SourceRepo;
 use Librecores\ProjectRepoBundle\Util\FileUtil;
-use Symfony\Component\Process\Process;
+use Librecores\ProjectRepoBundle\Util\MarkupToHtmlConverter;
+use Librecores\ProjectRepoBundle\Util\ProcessCreator;
+use Psr\Log\LoggerInterface;
 use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+use Symfony\Component\Process\Process;
```

Comment:
Why all those changes? Were they wrong before? I don't see changes in the code that would make those necessary.

Reply:
Some classes are necessary because I have created a constructor to inject projectmetrics along with all other classes